### DISCUSSION TITLE
Unit 4.1 Part 5 (slide 12)

### DISCUSSION THREAD

**Discussion by cornuch** [Lightning-AI/dl-fundamentals#39] [Lecture videos and quizzes] [answered]
Dear Sebastian Raschka,
Thank you for all that work.

I don't understand why the tensor is not ([[1,0,0],[0,0,1],[0,1,0],[0,1,0]]) instead of ([[1,0,0],[0,0,1],[0,0,1],[0,1,0]]).

**Comment by rasbt** [comment, marked as answer]
Hey there, the tensor is `([[1,0,0],[0,0,1],[0,0,1],[0,1,0]])` based on the class labels here in Slide 4. Does that help clarify? 
<img width="916" alt="Screenshot 2023-05-04 at 3 57 28 PM" src="https://user-images.githubusercontent.com/5618407/236327595-45af0dfc-4012-4b96-9810-0be1ad8c25ec.png">

**Reply by rasbt** [reply]
Marking it as answers to check which questions still need answering. Let me know in case you have any follow-up questions, and I'd be happy to chat more.

**Reply by TeemCh** [reply]
@rasbt He meant the following on the page 14:
![image](https://github.com/Lightning-AI/dl-fundamentals/assets/13614869/f03b0d16-ddc9-48ac-9728-6bc9fc3b35f7)
For the third training example the class membership should have been 1st index as it corresponds to higher probability 0.56
Best regards
